### DISCUSSION TITLE
Suggestion

### DISCUSSION THREAD

**Discussion by constantinsb** [007revad/Linux_Plex_Backup#11] [Ideas]
Thank you for making this script!

One thing though - I would recommend mentioning in the readme that you need to run "chmod +x" on either the script of choice or the whole directory, as without doing that you just get hit with a 'Permission Denied' error.

The other issue I'm having is that the script isn't finding my config-set backup directory, so I'm getting stonewalled there. Have you got any suggestions for what I might be doing wrong?

Edit to add: I can cd to the directory specified in the .config file from a terminal just fine. Even copy-pasted the directory directly.

Edit 2: I got the script working by removing the quotation marks from the Backup_Directory field, i.e. the line reads:
`Backup_Directory=/PlexMedia/Backups/Plex_Backup`
instead of 
`Backup_Directory="/PlexMedia/Backups/Plex_Backup"`

Not sure what that'll mean if there's any spaces in the filepath. I've also just confirmed that the same thing happens with the Name and LogAll fields.

**Comment by 007revad** [comment]
What Linux distro are you using?

I'll investigate why the double quotes are causing a problem and work a solution to prevent it.

I usually commit files to github with the executable attribute already set. I'll add "chown +x" instructions to the readme.

**Reply by constantinsb** [reply]
I'm using Ubuntu 24.04 LTS.

Thanks!

And it's entirely possible that the 'chown +x' is something implicit that I'm just too inexperienced to be doing automatically.

**Reply by 007revad** [reply]
I just tested this and without double quotes and file names and paths with spaces work.
```
root@DISKSTATION:~# /volume1/scripts/Linux_Plex_Backup.sh
BakDir: /share/Backups/Plex Backup
Name:   distro
LogAll: no
```

And with double quotes the double quotes are included, which will cause a problem.
```
root@DISKSTATION:~# /volume1/scripts/Linux_Plex_Backup.sh
BakDir: "/share/Backups/Plex_Backup"
Name:   "distro"
LogAll: "no"
```

In v1.0.5 I've changed the readme and conf file to remove the double quotes, and fixed Restore_Linux_Plex_Backup.sh which previously needed double quotes if there spaces.
https://github.com/007revad/Linux_Plex_Backup/releases

**Reply by constantinsb** [reply]
Thanks for the update!
Much appreciated.
